types/node/node-tests.ts

```diff
@@ -3075,11 +3063,11 @@ client.listbreakpoints((err, body, packet) => { });
 ////////////////////////////////////////////////////
 namespace async_hooks_tests {
     const hooks: async_hooks.HookCallbacks = {
-        init: (asyncId: number, type: string, triggerAsyncId: number, resource: object) => void {},
-        before: (asyncId: number) => void {},
-        after: (asyncId: number) => void {},
-        destroy: (asyncId: number) => void {},
-        promiseResolve: (asyncId: number) => void {}
+        init() {},
+        before() {},
+        after() {},
+        destroy() {},
+        promiseResolve() {},
```

Comment:
Isn't this removing the verification of the parameter types?

Reply:
I don't think it's useful to verify a declaration by simply repeating it .. tests should be there to verify that *using* the library works, not simply to assert that the declarations are what they are.